python/tvm/script/parser/core/entry.py

```diff
@@ -77,4 +95,15 @@ def parse(program: Union[doc.AST, Any, str], extra_vars: Dict[str, Any] = None)
             parser.parse(extra_vars=extra_vars)
         except ParserError as err:
             parser.report_error(err.node, err.args[0])
-    return builder.get()
+    ret = builder.get()
+    # well-formedness check will ignore any non-Relax functions
+    if isinstance(ret, IRModule):
```

Comment:
Can this be applied to a single relax function as well?  If the result is something other than an `IRModule`, we could construct `IRModule.from_expr(ret)`, and apply the well-formed check there.

Reply:
Sure, we could do it that way.